### PR TITLE
ModelBuilder API Doc Touchup

### PR DESCRIPTION
**Pull request by kjdelisle**
### Description
Quick nitpick fixes for the ModelBuilder API docs.

#### Related issues
connected to https://github.com/strongloop-internal/scrum-apex/issues/179

### Checklist
- [x] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style
  guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

File: lib/model-builder.js [hunk 1]
```diff
@@ -887,7 +892,7 @@ ModelBuilder.prototype.buildModels = function(schemas, createModel) {
  * @param {String} name The model name
  * @param {Object} json The JSON object
  * @param {Object} options The options
- * @returns {}
+ * @returns {ModelClass} The generated model definition object.
```
Comment (jannyHou) [inline lib/model-builder.js]: nitpick: I thought model definition object is part of the constructor but not a same concept?

Comment (kjdelisle) [inline lib/model-builder.js]: fixed!

File: lib/model-builder.js [hunk 2]
```diff
@@ -763,7 +763,10 @@ function hiddenProperty(where, property, value) {
 }
 
 /**
- * Get the schema name
+ * Get the schema name. If no parameter is given, then an anonymous model name
+ * is generated and returned.
+ * @param {string=} name The optional name parameter.
```
Comment (0candy) [inline lib/model-builder.js]: Is the `=` suppose to be there?

Comment (kjdelisle) [inline lib/model-builder.js]: The equals means the parameter is optional. :)

**Pull request merged**